sdk/data/azappconfig/client.go

```diff
@@ -275,3 +275,288 @@ func (c *Client) NewListSettingsPager(selector SettingSelector, options *ListSet
 		Tracer: c.appConfigClient.Tracer(),
 	})
 }
+
+// NewGetSnapshotsPager - Gets a list of key-value snapshots.
+//
+//   - options - NewGetSnapshotsPagerOptions contains the optional parameters to retrieve a snapshot
+//     method.
+func (c *Client) NewGetSnapshotsPager(options *NewGetSnapshotsPagerOptions) *runtime.Pager[ListGetSnapshotsPagerResponse] {
```

Comment:
`NewLilstSnapshotsPager` for the method name.

Reply:
do we also want to modify the options/Response as well as the generated code to make this change?

Reply:
Yes. I tagged all those separately.